VK/VK/CustomNavigationController/CustomInteractiveTransition.swift

```diff
@@ -0,0 +1,53 @@
+// CustomInteractiveTransition.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Custom animation closing ViewController
+final class CustomInteractiveTransition: UIPercentDrivenInteractiveTransition {
+    // MARK: - Public Properties
+
+    var viewController: UIViewController? {
+        didSet {
+            let recognizer = UIScreenEdgePanGestureRecognizer(
+                target: self,
+                action: #selector(handleScreenEdgeGestureAction(_:))
+            )
+            recognizer.edges = [.left]
+            viewController?.view.addGestureRecognizer(recognizer)
+        }
+    }
+
+    var isStarted: Bool = false
+    var isFinished: Bool = false
+
+    // MARK: - Public Methods
+
+    @objc func handleScreenEdgeGestureAction(_ recognizer: UIScreenEdgePanGestureRecognizer) {
```

Comment:
```suggestion
    @objc private func handleScreenEdgeGestureAction(_ recognizer: UIScreenEdgePanGestureRecognizer) {
```

я так посмотрел ты ни где не вызовешь  больше эту функцию так что думаю должна быть приватной